README.md

```diff
@@ -51,6 +51,10 @@ ServerImagekit imagekit = new ServerImagekit(publicKey, privateKey, urlEndPoint,
 
 **Note**: You can get the `apiKey`, `apiSecret`, and ImagekitId from your [Imagekit.io dashboard](https://imagekit.io/dashboard).
 
+## Demo application
+The fastest way to get started is by running the demo application in [Sample](/Sample) folder as per [README.md](/sample/README.md).
```

Comment:
There is not `Sample` folder on root. The sample app can be placed in `Sample` folder on the root with a `README.md` if needed.